src/daedalus_deploy/cli.py

```
@@ -111,3 +111,7 @@ def main(argv=None):
         constellation.obj.__getattribute__(action)(**args)
         if action == "stop" and args["remove_volumes"]:
             remove_config(path)
+        if action == "stop":
+            # stop container 'daedalus-grafana-alloy' using bash
+            os.system("docker stop daedalus-grafana-alloy")
```

Ruff (0.15.18)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)